lib/src/models/product.dart

```diff
@@ -9,7 +9,7 @@ class Product {
   /// Currency code for the price
   late String currencyCode;
   /// Local currency price for the product in double
-  late double price;
+  late num price;
```

Comment:
would this need any changes at the customer's logic?

Reply:
No impacts on customer logic.